dynamo/external/sctransform.py

```diff
@@ -0,0 +1,332 @@
+# =================================================================
+# Original Code Repository Author: Lause, Berens & Kobak
+# Adapted to Dynamo by: dynamo authors
+# Created Date: 12/16/2021
+# Description: pearson residuals based method for preprocessing single cell expression data
+# Original Code Repository: https://github.com/atarashansky/SCTransformPy
```

Comment:
sctransform is from seurat originally, right? cite seurat's paper / package, etc?

Reply:
Cited in preprocessor's preprocess_adata_sctransform method

Reply:
added to this file as well in new commits.